src/Controller/ContentElement/LegalTextElementController.php

```diff
@@ -84,7 +87,13 @@ protected function getResponse(Template $template, ContentModel $model, Request
         if (empty($html)) {
             // Show a message in the back end if no data for this legal text is available
             if ($this->scopeMatcher->isBackendRequest($request)) {
-                return new Response($this->translator->trans('data_not_available', ['%type%' => $model->er24Type], 'ContaoErecht24Rechtstexte'));
+                if($this->lastErecht24Error) {
+                    $errorTemplate = new FrontendTemplate('ce_er24_error');
+                    $errorTemplate->message = $this->lastErecht24Error;
+                    return new Response($errorTemplate->parse());
+                } else {
+                    return new Response($this->translator->trans('data_not_available', ['%type%' => $model->er24Type], 'ContaoErecht24Rechtstexte'));
+                }
```

Comment:
A custom template for a back end error is not really necessary imho, as the template in its current form only outputs the message (plus an additional wrapper, which is also not relevant for the back end). I think this should be changed so that the message is either the last API error if available, otherwise the general error from the translation.

Also, rather than saving the last error in a private variable I think it makes more sense to throw a custom exception within `getHtml()` which can then be captured here. This way the service is not polluted.

Reply:
Hello Fritz,
with the next version of the eRecht24 SDK we are going to provide a faq link, an error number and a better error message, this is why I used a template to have the ability to style this output in a future version.  
At the moment we still have the problem that the user does not see an error message in the backend, if this legal text was successfully fetched before.

Reply:
Ah I see, then it makes sense to be able to put this in a template yes. Though for consistency: templates intended for the back end are usually prefixed by `be_`, so it could be `be_er24_error.html5`. And it should be instantiated with the `BackendTemplate` class, rather than the front end template class.

When will the next SDK versions be released? May be it would make sense to wait until then and then also increase the composer dependency accordingly in one swoop.